Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -238,14 +246,33 @@ class TIFFLoadingContext {
         return Error::from_string_literal("Unsupported value for PhotometricInterpretation");
     }
 
+    ErrorOr<CMYK> read_color_cmyk(BigEndianInputBitStream& stream)
+    {
+        VERIFY(m_metadata.photometric_interpretation() == PhotometricInterpretation::CMYK);
+        auto bits_per_sample = *m_metadata.bits_per_sample();
+
+        auto const first_component = TRY(read_component(stream, bits_per_sample[0]));
+        auto const second_component = TRY(read_component(stream, bits_per_sample[1]));
+        auto const third_component = TRY(read_component(stream, bits_per_sample[2]));
+        auto const fourth_component = TRY(read_component(stream, bits_per_sample[3]));
+
+        // FIXME: Let's try to not drop alpha :)
```

Comment:
:thousandyakstare:

Is this a thing? CMYK is for accurate reproduction of colors on a printer, and printers can't do transparency. I suppose it's a thing in TIFF, yuck!

Reply:
TIFF does some stuff nicely but preventing you from using a weird combination of parameters is not one of them :sweat_smile:.